### PR TITLE
Add a data readme

### PR DESCRIPTION
**Pull request by aboucaud**
This PR aims at adding a description to the survey / filter parameters as well as the physical units they should be specified with.

### CODE REVIEW

**Comment by aboucaud** [comment]
@mpaillassa @HironaoMiyatake @ismael-mendoza could you please review ?

File: galcheat/data/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+#### `exp_time`
+
+Average exposure time of the filter on the same spot in the sky over the course of the survey.
+
+#### `psf_fwhm`
+
+Average full width at half-maximum (FWHM) of the point spread function (PSF) over the filter.
+
+This parameter is traditionally computed with the [`speclite`][speclite] library.
```
Comment (mpaillassa) [inline galcheat/data/README.md:77]: Hi @aboucaud, thanks for writing this readme ! I would just have one comment about the `psf_fwhm` description. I don't think we compute it with speclite, or did I miss something ?

Comment (aboucaud) [reply galcheat/data/README.md:77]: You're right, I got confused with the zeropoint.

Comment (mpaillassa) [reply galcheat/data/README.md]: Yes this what I thought ! Other than that I think it is fine !

**Comment by aboucaud** [comment]
Fixes #52

**Review by ismael-mendoza — APPROVED** [review]
Looks great @aboucaud! This looks pretty complete

File: galcheat/data/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+#### `obscuration`
+
+Proportion of the total area of the telescope that is obscured by the position of secondary mirrors, lenses, camera, etc.
+
+This parameter is used to compute the effective area of the telescope.
+
+#### `zeropoint_airmass`
+
+Airmass value at which the zeropoint is computed.
```
Comment (ismael-mendoza) [inline galcheat/data/README.md:46]: Could it be useful to add the definition of airmass ? "Optical path length through the atmosphere relative to the zenith path length." (from WeakLensingDeblending)

File: galcheat/data/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+-----------------
+### Units and types
+
+| parameter name       | type      | units          |
+| -------------------- | --------- | -------------- |
+| name                 | str       | –              |
+| sky_brightness       | float     | mag / arcsec^2 |
+| exposure_time        | int/float | s              |
+| psf_fwhm             | float     | arcsec         |
+| zeropoint            | float     | e- / s         |
```
Comment (ismael-mendoza) [inline galcheat/data/README.md:58]: Should the zeropoint be in `mag`? It's a magnitude not a flux right?

Comment (aboucaud) [reply galcheat/data/README.md:58]: Good catch, probably copy-pasted from the gain

Comment (aboucaud) [reply galcheat/data/README.md:58]: Nah it's because it's a magnitude for 1e-/s. And I got confused.

**Pull request merged**